src/App.js

```diff
@@ -84,20 +87,25 @@ export default function App() {
 
     async function postRequest(inputValue, inputLanguageValue, outputLanguageValue ){
         const params = new URLSearchParams({
-            input: inputValue,
-            in_lang: inputLanguageValue,
-            out_lang: outputLanguageValue
+          input: inputValue,
+          in_lang: inputLanguageValue,
+          out_lang: outputLanguageValue
         })
 
         const requestOptions = {
-            method: 'POST',
-            headers: new Headers({
-                'Content-Type': 'application/x-www-form-urlencoded',
-            }),
-            body: params.toString()
+          method: 'POST',
+          headers: new Headers({
+              'Content-Type': 'application/x-www-form-urlencoded',
+          }),
+          body: params.toString()
         };
         const response = await fetch('https://cjsback.herokuapp.com/', requestOptions);    
         const translation = await response.json();
+        
+        // change in_lang only if it is defined
+        if (typeof translation['response_in_lang'] !== "undefined"){
```

Comment:
Why not 
`
translation['response_in_lang'] !== undefined
`
?

Reply:
Just switched it - I did it using typeof because of a stack overflow answer but this works as well.